### PR TITLE
feat(Storybook): Light/dark mode switch

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Adds light/dark mode switch

Moreover:
- load asynchronously switch fixed
- storybook refactored to rely on build-in components only
- some explicit sizes/spacings removed
- some bigger components split into smaller

Closes: #7898

### Affected areas

`Storybook`

### Screenshot of functionality (including design for comparison)

Kazam_screenshot_00001

### CODE REVIEW

File: storybook/main.qml [hunk 1]
```diff
@@ -85,9 +105,10 @@ ApplicationWindow {
...
 
     Settings {
-        id: storeSettings
+        id: settings
 
-        property string selected: ""
-        property bool loadAsynchronously: false
+        property string currentPage: root.currentPage
+        property bool loadAsynchronously: loadAsyncCheckBox.checked
+        property bool nightMode: nightModeCheckBox.checked
```
Comment (caybro) [inline storybook/main.qml:112]: If you use `property alias ...` here, then you don't have to set the `checked` property on the nightMode checkbox :) Same for the other properties

**Review by caybro — APPROVED** [review]
Looks good, just one suggestion

File: storybook/main.qml (hunk 1 above)
Comment (caybro) [reply storybook/main.qml:112]: I mean, by using an `alias` in settings, it works both ways

Comment (micieslak) [reply storybook/main.qml:112]: Cool! I forgot abut that trick, thanks!

**Review by iurimatias — APPROVED** [review]

**Pull request merged**